Makefile.toml

```
@@ -266,9 +266,31 @@ cargo install \
 '''
 ]
 
+# We need Cargo version 1.51 or higher in order to build a workspace's transient dependency
+# during build-package
+[tasks.check-cargo-version]
+script_runner = "bash"
+script = [
+'''
+set -e
+cargo_version=$(cargo -V | awk -F '[() ]' '{print $2}')
```

```suggestion
cargo_version=$(cargo --version | awk '{print $2}')
```
Don't need the regex field separator since this version doesn't use the part in the parens.  (And `--version` to be a little clearer about what we're asking from cargo...)

Or if you want to simplify the next bits too:
```
IFS=. read -r major minor _ <<< "$(cargo --version | awk '{print $2}')"
```

---

I avoided `read -r` to stay compatible with old macbook bash. I wouldn't strive for that compatibility unless it was somewhat trivial to accomplish... but currently I think everything in Makefile works on stock/old bash that comes with 10.X.

I made the other changes.